CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [PR-86](https://github.com/salesforce/storm-dynamic-spout/pull/86) Sideline payload null checking
 - [PR-88](https://github.com/salesforce/storm-dynamic-spout/pull/88) Sideline payload null checking
 
+## 0.9.2 (2017-12-13)
```

Comment:
You’ll need to do a separate commit to master with the change log after this FYI.

Reply:
Going to just merge that change straight into master.